sdk/ml/azure-ai-ml/azure/ai/ml/_internal/_schema/component.py

```diff
@@ -52,6 +52,7 @@ class NodeType:
     # internal spake component got a type value conflict with spark component
     # this enum is used to identify its create_function in factories
     SPARK = "DummySpark"
+    AETHERGRIDEGE = "AetherBridgeComponent"
```

Comment:
Please add an test case of component yaml with this new type

Reply:
Add the test csae